.github/workflows/library_interop_tests.yml

```diff
@@ -0,0 +1,281 @@
+        working-directory: ./AwsEncryptionSDK/runtimes/net/TestVectorsNative/TestVectors
+        if: matrix.os != 'windows-latest'
+        shell: bash
+        run: |
+          PYTHON_23_VECTOR_PATH=$GITHUB_WORKSPACE/python23/vectors
+          if [ "$RUNNER_OS" == "macOS" ]; then
+            DAFNY_AWS_ESDK_TEST_VECTOR_MANIFEST_PATH="$PYTHON_23_VECTOR_PATH/manifest.json" \
+            DYLD_LIBRARY_PATH="/usr/local/opt/openssl@1.1/lib" \
+            dotnet test --framework net6.0
+          else
+            DAFNY_AWS_ESDK_TEST_VECTOR_MANIFEST_PATH="$PYTHON_23_VECTOR_PATH/manifest.json" \
+            dotnet test --framework net6.0
+          fi
+          
+  generate_vectors:
+    strategy:
+      matrix:
+        os: [
+          ubuntu-latest,
+          macos-latest,
+        ]
+    runs-on: ${{ matrix.os }}
+    permissions:
+      id-token: write
+      contents: read
+    env:
+      DOTNET_CLI_TELEMETRY_OPTOUT: 1
+      DOTNET_NOLOGO: 1
+    steps:
+      - name: Support longpaths on Git checkout
+        run: |
+          git config --global core.longpaths true
+      - uses: actions/checkout@v2
+      - name: Init Submodules
+        shell: bash
+        run: |
+          git submodule update --init libraries
+          git submodule update --init --recursive mpl
+          
+      - name: Configure AWS Credentials
+        uses: aws-actions/configure-aws-credentials@v4
+        with:
+          aws-region: us-west-2
+          role-to-assume: arn:aws:iam::370957321024:role/GitHub-CI-Public-ESDK-Dafny-Role-us-west-2 
+          role-session-name: NetTests
+      
+      - name: Setup .NET Core SDK 6
+        uses: actions/setup-dotnet@v3
+        with:
+          dotnet-version: '6.0.x'
+
+      - name: Setup Dafny
+        uses: dafny-lang/setup-dafny-action@v1.6.1
+        with:
+          dafny-version: ${{ inputs.dafny }}
+
+      - name: Download Dependencies 
+        working-directory: ./AwsEncryptionSDK
+        run: make setup_net
+
+      - name: Compile AwsEncryptionSDK implementation
+        shell: bash
+        working-directory: ./AwsEncryptionSDK
+        run: |
+          # This works because `node` is installed by default on GHA runners
+          CORES=$(node -e 'console.log(os.cpus().length)')
+          make transpile_net CORES=$CORES
+      
+      # - name: Generate Test Vectors with .NET Framework net6.0
+      #   # TODO Post-#619: Fix Zip file creation on Windows
+      #   if: matrix.os != 'windows-latest'
+      #   working-directory: ./AwsEncryptionSDK
+      #   shell: bash
+      #   run: |
+      #     NET_41_VECTOR_PATH=$GITHUB_WORKSPACE/net41/vectors
+      #     mkdir -p $NET_41_VECTOR_PATH
+      #     GEN_PATH=runtimes/net/TestVectorsNative/TestVectorGenerator
+      #     dotnet run --project $GEN_PATH --framework net6.0 -- \
+      #       --encrypt-manifest $GEN_PATH/resources/0006-awses-message-decryption-generation.v2.json \
+      #       --output-dir $NET_41_VECTOR_PATH
```

Comment:
Remove?

Reply:
I want to leave it to come back to fix the zip on windows problem